posix.mak

```diff
@@ -383,10 +383,16 @@ docs-prerelease: dmd-prerelease druntime-prerelease phobos-prerelease apidocs-pr
 
 docs : docs-latest docs-prerelease
 
-html : $(ALL_FILES) makebook
+html : html_files makebook
+
+html_files : $(ALL_FILES)
```

Comment:
Moving this into a different target ins't required but improves debugging the scripts.